packages/help-center/src/components/help-center-feedback-form.tsx

```diff
@@ -61,10 +58,6 @@ const HelpCenterFeedbackForm = ( { postId }: { postId: number } ) => {
 		generateContactOnClickEvent( 'chat', 'calypso_helpcenter_feedback_contact_support' );
 		if ( isUserEligibleForPaidSupport ) {
 			await resetSupportInteraction();
-			startNewInteraction( {
-				event_source: 'help-center',
-				event_external_id: uuidv4(),
-			} );
```

Comment:
We're removing this because `resetSupportInteraction` already start the new interaction, right?

Reply:
Exactly
On my local branch I went without not touching the interaction api which proved to be a mistake.
The calls need to be simplified like they are here, otherwise it gets out of hand